src/controllers/admin.js

```diff
@@ -1,17 +1,57 @@
-const { insertArticle } = require('../model/queries/addData');
+const {
+  insertArticle,
+  insertDoctor
+} = require('../model/queries/addData');
```

Comment:
you should first import others module then your modules and function so move this to be under bcrypt